.circleci/config.yml

```diff
@@ -2186,6 +2193,15 @@ jobs:
       - store_artifacts:
           path: /results.tar.gz
 
+  javadoc-generation:
+    executor:
+      name: build-executor
+    steps:
+      - run:
+          name: Generate javadocs
+          command: |
+            cd /repo; export JAVA_HOME="/usr/local/java/jdk-12.0.2"; mvn javadoc:javadoc
```

Comment:
Approve postmortem. I wonder why we have to set `JAVA_HOME` here. The docker image should have had it set up correctly, if not we should fix the docker image. Otherwise, every time we upgrade the JDK version, we have to manually change this.